Don't create /usr/include/dcbd and /usr/include/dcbd/clif_cmds.h in makefile. 
They will be created by debian/rules.
--- a/Makefile.am
+++ b/Makefile.am
@@ -94,11 +94,6 @@
 	$(MKDIR_P) $(DESTDIR)/etc/init.d
 	$(INSTALL_SCRIPT) lldpad.init $(DESTDIR)/etc/init.d/lldpad
 
-install-data-hook:
-	## provide legacy support for apps that use the old dcbd interface.
-	test -e $(includedir)/dcbd || `ln -T -s $(includedir)/lldpad $(includedir)/dcbd`
-	test -e $(includedir)/dcbd/clif_cmds.h || `ln -T -s $(includedir)/lldpad/lldp_dcbx_cmds.h $(includedir)/dcbd/clif_cmds.h`
-
 uninstall-local:
 	rm -f '$(DESTDIR)/etc/init.d/lldpad'
 	rm -f '$(includedir)/dcbd/clif_cmds.h'
